server/pages/index.js

```diff
@@ -800,20 +797,9 @@ module.exports = function() {
 			}
 		} );
 
-	function loginRouteSetup( req, res, next ) {
-		req.context = getDefaultContext( req );
-		const target = getBuildTargetFromRequest( req );
-		req.context.entrypoint = getFilesForEntrypoint( target, 'entry-login' );
-		setUpCSP( req, res, () =>
-			req.context.isLoggedIn
-				? setUpLoggedInRoute( req, res, next )
-				: setUpLoggedOutRoute( req, res, next )
-		);
-	}
-
```

Comment:
This functionality is present in the regular `setUpRoute()` function, so it should be fine to remove.